src/domains/Feedback/ReportForm/ReportForm.tsx

```diff
@@ -10,36 +10,19 @@ interface DefaultProps {
 }
 interface FormProps {
     title: string;
-    mainDescription: string;
-    icon: JSX.Element;
+    description: string;
     reportType: 'Feedback' | 'Bug';
     townhallId?: string;
 }
 
-export default function ReportForm({
-    title,
-    mainDescription,
-    icon,
-    reportType,
-    townhallId,
-}: FormProps & DefaultProps) {
+export default function ReportForm({ title, description, reportType, townhallId }: FormProps & DefaultProps) {
     return (
-        <Grid container spacing={5}>
+        <Grid container>
             <Grid item xs={12}>
-                <Typography variant='h4' align='center'>
-                    {title}
-                    {icon}
-                </Typography>
+                <FormTitle title={title} description={description} titleTypographyProps={{ align: 'center' }} />
```

Comment:
Are the grids here necessary with the report form here?

Reply:
No, removed.